src/borg/archiver.py

```diff
@@ -124,15 +124,16 @@ def decorator(method):
         def wrapper(self, args, **kwargs):
             location = args.location  # note: 'location' must be always present in args
             append_only = getattr(args, 'append_only', False)
+            storage_quota = getattr(args, 'storage_quota', None)
```

Comment:
getattr must be used here, because not every borg command has the `--storage-quota` argument.